src/containerapp/azext_containerapp/vendored_sdks/hybridkubernetes/__init__.py

```diff
@@ -0,0 +1,19 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from ._connected_kubernetes_client import ConnectedKubernetesClient
```

Comment:
is this file imported from other sdk or written by ourselves.

Reply:
The verdored sdk is the same as 'azure-mgmt-hybridkubernetes==1.1.0'.
 https://github.com/Azure/azure-cli-extensions/pull/7004#discussion_r1396628231